rest_api/actix_web_1/src/lib.rs

```diff
@@ -15,3 +15,6 @@
 #[macro_use]
 #[cfg(feature = "admin-service")]
 extern crate log;
+#[macro_use]
+#[cfg(feature = "admin-service")]
```

Comment:
This feature isn't in the cargo toml, nor are these dependencies optional

Reply:
fixed, reorderd commits to make sense and made dependencies optional